samples/tab-external-auth/csharp/assets/sample.json

```diff
@@ -0,0 +1,68 @@
+[
+  {
+    "name": "officedev-microsoft-tab-external-auth-csharp",
+    "source": "officeDev",
+    "title": "Tab External Auth (C#)",
+    "shortDescription": "Microsoft Teams sample app which showcases google authentication using external auth providers (C#).",
+    "url": "https://github.com/OfficeDev/Microsoft-Teams-Samples/tree/main/samples/tab-external-auth/csharp",
+    "longDescription": [
+      "This sample illustrates how to implement google authentication using external auth providers (C#)."
+    ],
+    "creationDateTime": "2023-08-16",
+    "updateDateTime": "2023-08-16",
+    "products": [
+      "Teams"
+    ],
+    "metadata": [
+      {
+        "key": "TEAMS-SAMPLE-SOURCE",
+        "value": "OfficeDev"
+      },
+      {
+        "key": "TEAMS-SERVER-LANGUAGE",
+        "value": "csharp"
+      },
+      {
+        "key": "TEAMS-SERVER-PLATFORM",
+        "value": "netframework"
+      },
+      {
+        "key": "TEAMS-FEATURES",
+        "value": "tab"
```

Comment:
External auth

Reply:
its related to capability like bot, ME, tab etc